00_responsive_web_design/03_technical_documentation/styles.css

```diff
@@ -0,0 +1,15 @@
+nav#navbar {
+    float: left;
+    width: 5em;
+}
```

Comment:
This could have a `position: fixed` attribute, so it stays in the window as the user scrolls.